pkg/omf/functions/packages/omf.packages.update.fish

```diff
@@ -4,15 +4,20 @@ function omf.packages.update -a name
     not test -e $target_path/.git;
       and return 0
 
-    if omf.repo.pull $target_path
-      omf.bundle.install $target_path/bundle
-      echo (omf::em)"$name successfully updated."(omf::off)
-      return 0
-    else
-      echo (omf::err)"Could not update $name."(omf::off) 1^&2
+    omf.repo.pull $target_path
+    switch $status
+      case 0
+        omf.bundle.install $target_path/bundle
+        echo (omf::em)"$name successfully updated."(omf::off)
+        return 0
+      case 1
+        echo (omf::err)"Could not update $name."(omf::off) 1>&2
```

Comment:
let's just `return 1` here rather than letting it fall back to the one at the bottom. it'll make it more clear.